### ISSUE TITLE
iziToast conflicting with iziModal

### ISSUE THREAD

**Issue by andtankian** [dolce/iziToast#23]
I've noticed iziToast title and message were fading out when called.
Then I changed the order of css links and it normalized, however iziModal assumed the behavior of iziToast.

Maybe same class used to both?

**Comment by dolce** [comment]
They are different classes. But I'll check.

**Comment by dolce** [comment]
They are different classes. But I'll check. thanks

**Comment by pablo-pacheco** [comment]
Same problem here

**Comment by dolce** [comment]
![captura de tela 2017-02-09 as 15 21 28](https://cloud.githubusercontent.com/assets/2837790/22794772/7d09e2c0-eedb-11e6-9402-363cf2604967.png)
I did the tests and it's normal, are you using the updated libraries?

**Comment by pablo-pacheco** [comment]
I'm using 1.4.2 for izimodal and 1.1.1 for easytoast.

If i change the order like @andtankian said i get izimodal fixed but izitoast gets weird. If i do the opposite izitoast gets fixed but izitoast breaks

I think the problem is css only because when I remove css from izitoast, izimodal gets fixed.

**Comment by dolce** [comment]
@pablo-pacheco, cara, me add no face e me explica melhor isso ae! 👍

**Comment by pablo-pacheco** [comment]
opa blz. Te procuro por marcelo dolce lá?

2017-02-09 15:48 GMT-02:00 Marcelo Dolce <notifications@github.com>:

**Comment by andtankian** [comment]
@dolce, in the image you've showed it's impossible to check.

I would recommend you to open a standard iziModal, not a customized one. Let it be toggled by a button for example.
Then setup an iziToast placing it's css link after the iziModal css link.

I've setup iziToast something like that:
```
iziToast.settings({
        timeout: 8000,
        transitionIn: "fadeInRight",
        transitionOut: "fadeOut",
        animateInside: true,
        pauseOnHover: false
});
```

When I open the iziModal, I get the behavior set to iziToast. (Title, icon and subtitle animating weirdly, like it's getting the configurations from iziToast)

**Comment by pablo-pacheco** [comment]
[This is a link to a gif](http://imgur.com/a/b9xoo) describing what is happening to me.

- This first image shows izimodal running sweet. 
- The second one is what happens when i load izitoast css after izimodal

**Comment by dolce** [comment]
I would like you to test this version of iziModal and give me a feedback.
https://www.dropbox.com/s/uvyufbbaom2noh3/iziModal_v1.5.0.zip

**Comment by pablo-pacheco** [comment]
Working smoothly now! Thanks!

**Comment by andtankian** [comment]
@dolce 
I've checked and now it is working as expected.
However, I would like to use bower to include this dependecy in my project.
Do you have any idea how long it will take to be released? v1.5.

Thanks.

**Comment by dolce** [comment]
@andtankian, I'll be updating iziToast and iziModal in the next few days. thanks
